tests/rptest/services/utils.py

```diff
@@ -135,7 +161,9 @@ def _check_oversized_allocations(self, line):
     def _search(self, nodes):
         bad_lines = collections.defaultdict(list)
         test_name = self._context.function_name
+        sw = Stopwatch()
         for node in nodes:
+            sw.start()
```

Comment:
Nit: may be creating a new timer per iteration is a simpler way?  I didn't realize before, but "StopWatch" logic is less simple than I thought.

Reply:
Why use 5 different Casios instead of pushing reset for new tracking? :)